openslides/assignment/forms.py

```diff
@@ -8,12 +8,12 @@
 
 
 class AssignmentForm(CssClassMixin, forms.ModelForm):
-    posts = forms.IntegerField(
+    member_count = forms.IntegerField(
```

Comment:
This fields means exactly what the description and the label says. When the assignment has 8 members from which 4 has to be elected, then member_count is 8.

Maybe we need another field "elected_count"?